openpype/modules/deadline/plugins/publish/validate_expected_and_rendered_files.py

```diff
@@ -12,18 +11,37 @@ class ValidateExpectedFiles(pyblish.api.InstancePlugin):
 
     label = "Validate rendered files from Deadline"
     order = pyblish.api.ValidatorOrder
-    families = ["render"]
     targets = ["deadline"]
 
     # check if actual frame range on render job wasn't different
     # case when artists wants to render only subset of frames
     allow_user_override = True
 
     def process(self, instance):
-        self.instance = instance
-        frame_list = self._get_frame_list(instance.data["render_job_id"])
+        """Process all the nodes in the instance"""
+        self.log.debug("Validate expected files for instance")
```

Comment:
Just logging that the plugin is running is redundant - if we need to know if a plugin runs and we can't from the UX or the publishing system - then that's an issue with the publisher itself and its debugging, not with the plug-in. We shouldn't need to log a "plugin is running" statement like this unless we're actually logging information that is unique to what is going on.
```suggestion
```